packages/cf-component-form/src/FormHeader.js

```diff
@@ -1,19 +1,23 @@
 import React, { PropTypes } from 'react';
+import { Heading } from 'cf-component-heading';
+import { createComponent } from 'cf-style-container';
 
-class FormHeader extends React.Component {
-  render() {
-    return (
-      <div className="cf-form__header">
-        <h3 className="cf-form__title">
-          {this.props.title}
-        </h3>
-      </div>
-    );
-  }
-}
+const styles = ({ theme }) => ({
+  padding: theme.padding,
+  borderBottom: theme.borderBottom
+});
+
+const FormHeader = ({ className, title }) => (
+  <div className={className}>
+    <Heading size={4}>
```

Comment:
Done. I still used h3 since it matches the old styles. We will redesign all forms pretty soon anyway (more info coming soon).